### PR TITLE
ODIM-3867 Add fsnotify logic in lib-utilities 

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-aggregation/go.mod [hunk 1]
```diff
@@ -5,10 +5,11 @@ go 1.13
...
 	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201007090649-06a4c52ce4cf
-	github.com/ODIM-Project/ODIM/lib-persistence-manager v0.0.0-20201021053518-88345d4a9988 // indirect
 	github.com/ODIM-Project/ODIM/lib-rest-client v0.0.0-20201007090649-06a4c52ce4cf
 	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201016064732-c652b5935f93
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/satori/go.uuid v1.2.0
 	github.com/stretchr/testify v1.5.1
 )
+
+replace github.com/ODIM-Project/ODIM/lib-utilities => ../lib-utilities
```
Comment (jeevan-kamkar) [inline svc-aggregation/go.mod]: why this changes required

File: svc-aggregation/agcommon/common.go [hunk 2]
```diff
@@ -159,41 +158,13 @@ func (e *DBInterface) AddConnectionMethods(connectionMethodConf []config.Connect
 // TrackConfigFileChanges monitors the odim config changes using fsnotfiy
 // Whenever  any config file changes and events  will be  and  reload the configuration and verify the existing connection methods
 func TrackConfigFileChanges(dbInterface DBInterface) {
-	watcher, err := fsnotify.NewWatcher()
-	if err != nil {
```
Comment (jeevan-kamkar) [inline svc-aggregation/agcommon/common.go]: go mod changes referred with lib utilities don't remove these changes

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the changes

File: svc-aggregation/agcommon/common.go (hunk 2 above)
Comment (irappa-pattar) [reply svc-aggregation/agcommon/common.go]: addressed

File: svc-aggregation/go.mod (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/go.mod]: addressed

**Review by jeevan-kamkar — APPROVED** [review]

**Review by skhan-anjum — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

**Pull request merged**